changelog/3083.feature.rst

```diff
@@ -0,0 +1 @@
+Add a helper utility for creating a `sunpy.map.Map` header
```

Comment:
How about:
```suggestion
Added a helper function that allows users to create a meta header for custom created `sunpy.map.Map`.
```

I think we want to expand on what its for and just to hopefully make it stand out in the changelog!